lib/chef/azure/commands/enable.rb

```diff
@@ -112,31 +113,41 @@ def configure_chef_only_once
 
       load_settings
 
-      # Write validation key
-      File.open("#{bootstrap_directory}/validation.pem", "w") do |f|
-        f.write(@validation_key)
-      end
-
-      # Write client.rb
-      File.open("#{bootstrap_directory}/client.rb", "w") do |f|
-        f.write(override_clientrb_file(@client_rb))
-      end
-      # write the first_boot.json
-      File.open("#{bootstrap_directory}/first-boot.json", "w") do |f|
-        f.write(<<-RUNLIST
-{
-"run_list": [#{ @run_list.empty? ? "" : escape_runlist(@run_list)}]
-}
-RUNLIST
-)
-      end
-
       # run chef-client for first time with no runlist to register the node
       puts "Running chef client for first time with no runlist..."
 
       begin
-        params = " -c #{bootstrap_directory}/client.rb -E _default -L #{@azure_plugin_log_location}/chef-client.log --once "
-        result = shell_out("chef-client #{params}")
+        require 'chef/azure/core/bootstrap_context'
+        config = {}

```

Comment:
This appears to be where we've added new json options to the extension. We should update the README to document the json format.